ppx/ppx_lwt_ex.ml

```diff
@@ -337,18 +344,18 @@ let lwt_mapper args =
         else
           default_mapper.expr mapper expr
       | { pexp_desc = Pexp_apply (fn, args); pexp_attributes; pexp_loc } when !log ->
-        default_loc := pexp_loc;
+        default_loc := ghost pexp_loc;
```

Comment:
It's mapping an expression from the input tree, it is a bit too early to set it ghost.